tests/runtime/watchpoint

```diff
@@ -56,6 +56,6 @@ REQUIRES_FEATURE signal
 
 NAME unique_probe_bodies
 RUN {{BPFTRACE}} -e 'watchpoint:increment_*+arg0:4:w { printf("%s!\n", probe) }' -c ./testprogs/watchpoint_func_wildcard
-EXPECT_REGEX .*increment_0:4:w!
+EXPECT_REGEX .*increment_0:4:w..!
```

Comment:
I think because the probe, now a string type, exceeds the default max_strlen. TBH, I gotta hunt this down but it was confusing the heck out of me.

Reply:
Ok, this is because how PrintableString works wih max_strlen. Maybe we should auto increase the max_strlen if the max probe length is larger. I'll have to think on this a bit more but let me know if it should be a blocker.

Reply:
Definitely not a blocker, I was just curious why that happened.